bootstrap.sh

```diff
@@ -0,0 +1,39 @@
+# Set the path for the file we are going to generate
+OUTPUT="$(pwd)/setup.txt"
+
+# Figure out what OS we are on
+current_os="$(uname -s)"
+case "${current_os}" in
```

Comment:
This and the case statement below are pretty identical but I wanted a separate straight forward variable that represented the `os` rather than putting all the os specific logic there as well

Reply:
Makes sense!